ros/lib/constants.py

```diff
@@ -27,3 +27,8 @@ class SubStates(Enum):
 class CloudProvider(Enum):
     """Supported cloud providers"""
     AWS = 'aws'
+
+
+class OperatingSystem(Enum):
```

Comment:
It does not seem we have plan to support other Operating Systems? so do we need Enum class? I absolutely understand keeping the implementation similar to other constants, however having question as well?

Reply:
BUs are still discussing. So, it is fine to keep Enum.
You can refer to [this](https://redhat-internal.slack.com/archives/C01C1GVT3TP/p1693230168469569?thread_ts=1693223068.533649&cid=C01C1GVT3TP) slack thread.